app/register/index.js

```diff
@@ -43,23 +44,47 @@ const activatePasswordValidation = () => {
 		}
 
 		const regExCheckNumber = /([0-9])/;
-		validatePasswordCharacters(regExCheckNumber, 'number')(event);
+		validateInput(regExCheckNumber, 'number')(event);
 
 		const regExCheckLowercase = /([a-z])/;
-		validatePasswordCharacters(regExCheckLowercase, 'lowercase')(event);
+		validateInput(regExCheckLowercase, 'lowercase')(event);
 
 		const regExCheckUppercase = /([A-Z])/;
-		validatePasswordCharacters(regExCheckUppercase, 'uppercase')(event);
+		validateInput(regExCheckUppercase, 'uppercase')(event);
 	};
 
 	passwordElement.addEventListener('change', validatePassword);
 };
 
+const activateEmailValidation = () => {
+	const validateEmail = (event) => {
+		// eslint-disable-next-line no-useless-escape
+		const regExCheckEmail = /@[a-zA-Z\-\.]+/g;
+		validateInput(regExCheckEmail, 'valid-email')(event);
+	};
+
+	emailElement.addEventListener('change', validateEmail);
```

Comment:
Interestingly, there is a lot of complication to do with what constitutes a valid email address and what doesn't.

https://stackoverflow.com/questions/46155/how-to-validate-an-email-address-in-javascript

Additionally, there's also the line of using the email input type (but I think it is probably less useful here considering we're doing validation of other fields too)

Reply:
Added a more thorough regex.